runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -278,7 +278,7 @@ public class LootTrackerPlugin extends Plugin
 
 	private static final String WINTERTODT_SUPPLY_CRATE_EVENT = "Supply crate (Wintertodt)";
 	private static final String WINTERTODT_REWARD_CART_EVENT = "Reward cart (Wintertodt)";
-	private static final String WINTERTODT_LOOT_STRING = "You found some loot: ";
+	private static final Pattern WINTERTODT_LOOT_PATTERN = Pattern.compile("(<col=ef1020>)?You found some loot: ");
```

Comment:
The optional leading color tag does nothing, because the "You found some loot: " string would still be matched. See: the burnt pages test you added which uses `<col=af1020>` (**a**, not *e*) which still passes.

Reply:
Generally instead of using fixed hex colors we will match `[0-9a-f]{6}` as these sometimes differ between opaque and transparent chat backgrounds.